packages/webview_flutter/lib/src/webview_android.dart

```diff
@@ -50,7 +50,7 @@ class AndroidWebView implements WebViewPlatform {
         // WebView content is not affected by the Android view's layout direction,
         // we explicitly set it here so that the widget doesn't require an ambient
         // directionality.
-        layoutDirection: TextDirection.rtl,
+        layoutDirection: TextDirection.ltr,
```

Comment:
Looks like this doesn't take "Directionality.of(context)" into account. See https://github.com/flutter/flutter/issues/66208#issuecomment-738247198